pkg/cmd/root/root.go

```diff
@@ -125,18 +125,12 @@ func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
 	cmd.AddCommand(labelCmd.NewCmdLabel(&repoResolvingCmdFactory))
 
 	// Help topics
-	cmd.AddCommand(NewHelpTopic(f.IOStreams, "environment"))
-	cmd.AddCommand(NewHelpTopic(f.IOStreams, "formatting"))
-	cmd.AddCommand(NewHelpTopic(f.IOStreams, "mintty"))
-	cmd.AddCommand(NewHelpTopic(f.IOStreams, "exit-codes"))
-	referenceCmd := NewHelpTopic(f.IOStreams, "reference")
-	referenceCmd.SetHelpFunc(referenceHelpFn(f.IOStreams))
-	cmd.AddCommand(referenceCmd)
+	for _, ht := range HelpTopics {
```

Comment:
One nice thing about moving this into a slice of `helpTopic` is that we've reduced the amount of knowledge that existed in two places i.e. the names of the help topics themselves. Similar to my comment above, it's nice to know that if something is in the slice of `HelpTopics`, it will be in the command list and counter to that, there's no way to reference a meaningless help topic here.